network/src/peers/peers.rs

```diff
@@ -71,7 +71,7 @@ impl Node {
             let number_to_connect = crawling_capacity.saturating_sub(active_peer_count - number_to_disconnect);
 
             (number_to_disconnect, number_to_connect)
-        } else if self.config.is_bootnode() {
+        } else if self.config.is_sync_provider() || self.config.is_beacon() {
             // Bootnodes disconnect down to 80% of their max to leave capacity open for new
```

Comment:
nit: you'll probably need an `rg -i` pass for the `bootnode` phrase :microscope: